src/probes/README.md

```diff
@@ -0,0 +1,53 @@
+# Probes
+
+The default probes provided along the telemetry client code are:
+
+* bertprobe: this probe reports on the Boot Error Region Table if such
+  entry is found in ```/sys/firmware/acpi/tables/BERT```.
+
+* crashprobe: this probe collects the core dump file from the kernel. This
+  probe is meant to be 'configured' in ```/proc/sys/kernel/core_pattern```.
```

Comment:
I would describe the crashprobe like so: "This probe processes core dump files. It can be registered as the kernel core file handler in /proc/sys/kernel/core_pattern."